lib/clients/redis.js

```diff
@@ -0,0 +1,48 @@
+var util = require('util');
+var BaseClient = require('./base');
+var _ = require('lodash');
+
+function buildRedisKey(namespace, metricName) {
+  return [namespace, metricName].join(':');
+}
+
+/**
+ * A client that can send metrics data to a redis backend
+ * @param {Object} opts 
+ * @param {Object} opts.redisClient an instance of the redis nodejs client.
+ * @param {Number} opts.recordsToKeep the number of records to keep for each metric. Default is 1000.
+ * @param {String} opts.namespace name space for each metric key. It will be used to prefix each metric key followed by ':'. Default is `stats`. So each metric key will be like `stats:<metric key>`.
+ */
+function RedisClient(opts) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'RedisClient' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)